### ISSUE TITLE
CSLA NuGet source code preview and debugging.

### ISSUE THREAD

**Issue by rxelizondo** [MarimerLLC/cslaforum#169]
Hello,

Out of curiosity, is it possible to view the source code and debug CSLA dlls installed from NuGet? I never done something like that but I read somewhere that it is possible provided that the publisher supports it.

Just curious.

Thanks.

**Comment by rockfordlhotka** [comment]
Yes, we do upload the code to symbol server every time we push the assemblies to nuget. We use the same symbol server as everyone else (including Microsoft afaik), so you should be able to find instructions on how to enable the use of the symbol server within the VS debugger via search.

**Comment by rxelizondo** [comment]
Thanks Rocky,

For some reason, I am unable to get this to work. When I look at the module information for the CSLA.dll file it shows "Cannot find or open the PDB file".

Could you let me know the specific symbol server that you guys are using to publish the source code? Is it the same one that comes set as default on Visual Studio 2015 (Microsoft Symbol Server)?

Any other tips to get this working are highly appreciated!

Thanks.
